portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Policies/PolicyForm/UploadPolicyDropzone.tsx

```diff
@@ -105,10 +106,23 @@ const UploadPolicyDropzone: FC<UploadPolicyDropzoneProps> = ({
     setPolicyDefinitionFile,
     gateway,
 }) => {
-
+    const intl = useIntl();
 
     const handleDrop = (policyDefinition: any) => {
-        setPolicyDefinitionFile(policyDefinition);
+        if(policyDefinition===null || policyDefinition===undefined) {
```

Comment:
Formatting issues

Reply:
And also this can be simplified as follows.

```js
const handleDrop = (policyDefinition: any) => {
    if (policyDefinition && policyDefinition[0] && (policyDefinition[0].name.endsWith('.j2') || policyDefinition[0].name.endsWith('.xml'))) {
        setPolicyDefinitionFile(policyDefinition);
        return;
    }
    APIMAlert.error(intl.formatMessage({
        id: 'Uploading.Policies.Error',
        defaultMessage: 'Incompatible file type',
    }));
};
```